src/core/api/verifications/verifications.types.ts

```diff
@@ -27,7 +27,7 @@ export interface VerificationRes extends VerificationReq {
   connection_url?: string;
   connection_id?: string;
   present_id?: string;
-  status: 'REQUESTED' | 'VEIFIED' | 'FAILED';
```

Comment:
Did you check it with Ehsan to change in BE as well?

Reply:
let me check it

Reply:
I changed it to previous version because we used `VEIFIED` in many places and I'm not sure about it. so please check it yourself with Ehsan.